sql/updates/world/master/2025_99_99_99_wandering_isle_quest_29664.sql

```diff
@@ -0,0 +1,91 @@
+-- Quest
+UPDATE `quest_template_locale` SET `LogTitle`='Las llamas del desafío', `LogDescription`='Enciende tu antorcha de contendiente sin encender en el blandón de llamas chisporroteantes y luego úsala para encender los 3 blandones cerca del maestro Li Fei.', `QuestDescription`='Tu mente y tu voluntad son fuertes, pero te falta la habilidad necesaria para desafiarme en combate. Debes luchar con el cuerpo, pero en realidad es tu carácter lo que está a prueba.$b$bVamos a darte un poco de ventaja.$b$bCoge esta antorcha, enciéndela en el blandón que tengo al lado y luego enciende los tres blandones que hay alrededor de la charca.', `VerifiedBuild`=55461 WHERE (`ID`=29664 AND `locale`='esES');
+
+DELETE FROM `quest_offer_reward_locale` WHERE (`locale`='esES' AND `ID` IN (29664,29420));
+INSERT INTO `quest_offer_reward_locale` (`ID`, `locale`, `RewardText`, `VerifiedBuild`) VALUES
+(29664, 'esES', 'Esas llamas te darán la fuerza que aún no has obtenido. Iluminarán tu potencial.', 55461),
+(29420, 'esES', 'El objetivo de la vida no es encontrarse a uno mismo: es crearse a uno mismo. Un camino se extiende ante ti, pero eres tú quien debe elegir las pruebas a las que te enfrentarás... y las pruebas que superarás.', 55461);
+
+DELETE FROM `quest_template_addon` WHERE `ID` IN (29420, 29664, 29421);
+INSERT INTO `quest_template_addon` (`ID`, `MaxLevel`, `AllowableClasses`, `SourceSpellID`, `PrevQuestID`, `NextQuestID`, `ExclusiveGroup`, `RewardMailTemplateID`, `RewardMailDelay`, `RequiredSkillID`, `RequiredSkillPoints`, `RequiredMinRepFaction`, `RequiredMaxRepFaction`, `RequiredMinRepValue`, `RequiredMaxRepValue`, `ProvidedItemCount`, `SpecialFlags`, `ScriptName`) VALUES 
+(29420, 0, 0, 0, 29418, 29664, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, ''),
+(29664, 0, 0, 0, 29420, 29421, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 0, ''),
+(29421, 0, 0, 0, 29664, 29422, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, '');
```

Comment:
NextQuest is used together with ExclusiveGroups to unlock the "next" quest.
if you want to unlock a quest which needs 3 Quests done, you need to put the 3 Quests into a negative exclusiveGroup and put the Quest into NextQuest which is the quest you unlock.